src/teaching-element/Audio.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="te-audio">
+    <aplayer
+      v-if="url"
+      v-bind="playerOptions"
+      :music="music" />
+  </div>
+</template>
+
+<script>
+import Aplayer from 'vue-aplayer';
+
+Aplayer.disableVersionBadge = true;
+const defaultAudioOptions = () => ({
+  title: 'Audio track',
+  artist: ' ',
+  pic: ' '
+});
+
+export default {
+  name: 'te-audio',
+  inheritAttrs: false,
+  props: {
+    url: { type: String, required: true },
+    audioOptions: { type: Object, default: defaultAudioOptions },
+    playerOptions: { type: Object, default: () => ({}) }
```

Comment:
`tailor-teaching-elements` is one composite component and is consumed as such which means that all of the props (coming from outside) are shared and you have the same prop as `te-video` (`playerOptions`) which makes it hard for the consumer to maintain them.

Reply:
Come to think about it, maybe we should make `playerOptions` and `audioOptions` as one prop.

Reply:
> `tailor-teaching-elements` is one composite component and is consumed as such which means that all of the props (coming from outside) are shared and you have the same prop as `te-video` (`playerOptions`) which makes it hard for the consumer to maintain them.

I don't understand what is your point here.  Consumer doesn't maintain anything he passes down the props which are being used. He can name data for that prop what ever he want and pass it down to `te-audio` or `te-video` as `playerOptions` which are isolated from each other. 

Let's not complicate things.

Reply:
> Come to think about it, maybe we should make `playerOptions` and `audioOptions` as one prop.

There not the same type of options and they should stay separated. `audioOptions` are options for audio files (title, pic, artist), `playerOptions` are player options. Check [the docs](https://github.com/SevenOutman/vue-aplayer/blob/develop/docs/README.md).

Reply:
> `tailor-teaching-elements` is one composite component and is consumed as such which means that all of the props (coming from outside) are shared and you have the same prop as `te-video` (`playerOptions`) which makes it hard for the consumer to maintain them.

I'll name it `audioPlayerOptions`.